Makefile

```diff
@@ -43,8 +46,8 @@ copy: $(RUNTIME) $(DISK_IMAGE)
 	rm -rf $(MOUNT_DIR)
 
 $(RUNTIME): $(ASM_OBJS) $(OBJS) $(SDK_EDGE_LIB) $(TMPLIB)
-	$(LINK) $(LINKFLAGS) -o $@ $^ -T runtime.lds
-	$(OBJCOPY) --add-section .options_log=.options_log --set-section-flags .options_log=noload,readonly $(RUNTIME)
+	$(LINK) $(LINKFLAGS) -o $@ $^ -T runtime.lds $(RISCV)/lib/gcc/riscv$(BITS)-unknown-elf/8.3.0/libgcc.a
```

Comment:
Why are we adding linking to libgcc here? I don't think we used to end up with it linked it.
Was this needed for something in rv32 support?

Reply:
Specifically for rv32, our QEMU version doesn't have hardware support for operations on "long long", so rv32 requires libgcc -- specifically __umoddi3 and __udivdi3.

Reply:
OK, we should make a note about this somewhere and file a TODO. Long term we don't want to have to link in libgcc... at least not entirely.